src/device/mod.rs

```diff
@@ -238,7 +245,7 @@ pub struct Entities {
     firewall: Arc<StatefullFirewall>,
 
     // Entities for meshnet connections
-    meshnet: Option<MeshnetEntites>,
+    meshnet: Either<MeshnetEntites, MeshnetEntitiesLastState>,
```

Comment:
`Either` introduces concepts of `left` and `right` which is not as readable as if it would be an enum IMHO, though the idea of `meshnet` having either entities or last state is very nice in itself even though unexpected(would except it to be saved separately and used as needed) :)

Reply:
> `Either` introduces concepts of `left` and `right` which is not as readable as if it would be an enum IMHO, though the idea of `meshnet` having either entities or last state is very nice in itself even though unexpected(would except it to be saved separately and used as needed) :)

I wanted to avoid a case when it's ambiguous which one to use - entites or the saved state. This way it's impossible to have access to both at the same time.

I will replace either with custom enum with domain specific names.

Reply:
Done.